Mage.Sets/src/mage/sets/MysteryBooster.java

```diff
@@ -0,0 +1,3704 @@
+        cards.add(new SetCardInfo("Evra, Halcyon Witness", 97, Rarity.RARE, mage.cards.e.EvraHalcyonWitness.class));
+        cards.add(new SetCardInfo("Excavation Elephant", 98, Rarity.COMMON, mage.cards.e.ExcavationElephant.class));
+        cards.add(new SetCardInfo("Exclude", 369, Rarity.UNCOMMON, mage.cards.e.Exclude.class));
+        cards.add(new SetCardInfo("Excoriate", 99, Rarity.COMMON, mage.cards.e.Excoriate.class));
+        cards.add(new SetCardInfo("Executioner's Capsule", 650, Rarity.COMMON, mage.cards.e.ExecutionersCapsule.class));
+        cards.add(new SetCardInfo("Expedite", 918, Rarity.COMMON, mage.cards.e.Expedite.class));
+        cards.add(new SetCardInfo("Expedition Raptor", 100, Rarity.COMMON, mage.cards.e.ExpeditionRaptor.class));
+        cards.add(new SetCardInfo("Experiment One", 1201, Rarity.UNCOMMON, mage.cards.e.ExperimentOne.class));
+        cards.add(new SetCardInfo("Explore", 1202, Rarity.COMMON, mage.cards.e.Explore.class));
+        cards.add(new SetCardInfo("Explosive Vegetation", 1203, Rarity.UNCOMMON, mage.cards.e.ExplosiveVegetation.class));
+        cards.add(new SetCardInfo("Expose Evil", 101, Rarity.COMMON, mage.cards.e.ExposeEvil.class));
+        cards.add(new SetCardInfo("Expropriate", 370, Rarity.MYTHIC, mage.cards.e.Expropriate.class));
+        cards.add(new SetCardInfo("Exsanguinate", 651, Rarity.UNCOMMON, mage.cards.e.Exsanguinate.class));
+        cards.add(new SetCardInfo("Extract from Darkness", 1424, Rarity.UNCOMMON, mage.cards.e.ExtractFromDarkness.class));
+        cards.add(new SetCardInfo("Exultant Skymarcher", 102, Rarity.COMMON, mage.cards.e.ExultantSkymarcher.class));
+        cards.add(new SetCardInfo("Eyeblight's Ending", 652, Rarity.COMMON, mage.cards.e.EyeblightsEnding.class));
+        cards.add(new SetCardInfo("Eyes in the Skies", 103, Rarity.COMMON, mage.cards.e.EyesInTheSkies.class));
+        cards.add(new SetCardInfo("Ezuri's Archers", 1204, Rarity.COMMON, mage.cards.e.EzurisArchers.class));
+        cards.add(new SetCardInfo("Fact or Fiction", 371, Rarity.UNCOMMON, mage.cards.f.FactOrFiction.class));
+        cards.add(new SetCardInfo("Fade into Antiquity", 1205, Rarity.COMMON, mage.cards.f.FadeIntoAntiquity.class));
+        cards.add(new SetCardInfo("Faerie Conclave", 1666, Rarity.UNCOMMON, mage.cards.f.FaerieConclave.class));
+        cards.add(new SetCardInfo("Faerie Invaders", 372, Rarity.COMMON, mage.cards.f.FaerieInvaders.class));
+        cards.add(new SetCardInfo("Faerie Mechanist", 373, Rarity.COMMON, mage.cards.f.FaerieMechanist.class));
+        cards.add(new SetCardInfo("Failed Inspection", 374, Rarity.COMMON, mage.cards.f.FailedInspection.class));
+        cards.add(new SetCardInfo("Faith's Fetters", 105, Rarity.COMMON, mage.cards.f.FaithsFetters.class));
+        cards.add(new SetCardInfo("Faithbearer Paladin", 104, Rarity.COMMON, mage.cards.f.FaithbearerPaladin.class));
+        cards.add(new SetCardInfo("Faithles
```

Comment:
`CardRepository.instance.findCard(setCode, cardNumber)` takes the _collector number_ and not the card name.
Use  `CardRepository.instance.findCardWPreferredSet(cardName, setCode, caseInsensitive)` instead